packages/grafana-ui/src/slate-plugins/braces.ts

```diff
@@ -2,7 +2,7 @@ import { Annotation } from 'slate';
 import { Plugin } from 'slate-react';
 import { v4 as uuidv4 } from 'uuid';
 
-const BRACES: any = {
+const BRACES: Record<string, string> = {
```

Comment:
this is to force us to build a canary version when this is merged to check everything has worked ok 👌